CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Mapbox welcomes participation and contributions from everyone.
 ### Changelog
 [Changes between v2.8.2 and v2.8.3](https://github.com/mapbox/mapbox-navigation-android/compare/v2.8.2...v2.8.3)
 
+#### Bug fixes and improvements
+- Fixed some internal Copilot-related issues and added minor improvements.
```

Comment:
PR link?

Reply:
There were some but opted to not link as they were internal.